### PR TITLE
Add hg-evolve/meta.yaml

### PR DESCRIPTION
**Pull request by paugier**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/hg-evolve/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7ef633c1ab9ef4ef9da8c74165a7b12daa32a38d6b3de9324ba4ce2b42ea126
+
+build:
+  number: 0
+  skip: True  # [py3k]
```
Comment (chrisburr) [inline recipes/hg-evolve/meta.yaml]: Skip can't be used with `noarch` packages but you can apply the Python version constraint in a different way.
```suggestion
```

File: recipes/hg-evolve/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (chrisburr) [inline recipes/hg-evolve/meta.yaml]:
```suggestion
    - python <3
```

File: recipes/hg-evolve/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - python
```
Comment (chrisburr) [inline recipes/hg-evolve/meta.yaml]:
```suggestion
    - python <3
```

File: recipes/hg-evolve/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+    - hgext3rd
+    - hgext3rd.evolve
+    - hgext3rd.evolve.thirdparty
+    - hgext3rd.topic
+
+about:
+  home: https://www.mercurial-scm.org/doc/evolution/
+  license: GPLv2+
+  license_family: GPL2
```
Comment (chrisburr) [inline recipes/hg-evolve/meta.yaml]:
```suggestion
  license_file: COPYING
```

**Comment by paugier** [comment]
@chrisburr Thanks a lot for your comments!

File: recipes/hg-evolve/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python < 3
```
Comment (chrisburr) [inline recipes/hg-evolve/meta.yaml]: This has to be formatted like this else some parts of conda-forge have issues rendering the recipe:
```suggestion
    - python <3
```
(one space before the operator, no spaces after)

File: recipes/hg-evolve/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python < 3
```
Comment (chrisburr) [inline recipes/hg-evolve/meta.yaml]: Same here
```suggestion
    - python <3
```

**Comment by paugier** [comment]
I modified the `< 3` as `<3` (nearly ❤️) 

> As this is noarch I think you can ignore the Windows failure as it will only be built on Linux in the feedstock.

Is there anything more I have to do now?

**Comment by chrisburr** [comment]
Nope, I just need to be reminded to check/merge this 😄

**Pull request merged**